src/main/java/subway/stations/ClosestStation.java

```diff
@@ -0,0 +1,35 @@
+package subway.stations;
+
+import subway.stations.SubwayStations.Station;
+import java.util.List;
+
+public class ClosestStation {
+
+    public Station getClosestStation( List<Double> coordinates, SubwayStations subwayStations){
+
+        Station station;
+
+        double shortestDistance = Double.MAX_VALUE;
+        int closestStation = 0;
+
+        for(int ix = 0; ix < subwayStations.stations.size(); ix++){
```

Comment:
Use a for-each loop and save the Station that is closest, not the index of the station.